tests/integration/api_v3/http/rewardInflation.test.js

```diff
@@ -29,16 +29,21 @@ const {
 
 const endpoint = `${baseUrlV3}/reward/inflation`;
 
-describe('Reward Inflation API', () => {
+describe('Test Reward Inflation API', () => {
 	describe(`GET ${endpoint}`, () => {
-		it('Returns current inflation rate', async () => {
-			const response = await api.get(endpoint);
+		it('should return current inflation rate when called with block height=1', async () => {
+			const response = await api.get(`${endpoint}?height=1`);
```

Comment:
Instead of hard-coding the `height=1` in the request query, let us query for the latest block height in the `beforeAll` hook and use that instead. `height: 1` won't exist on testnet/mainnet.

Reply:
Applied. But in general, do we have plan to run test against any other net than devnet?

Reply:
Yes, we run the tests against the mainnet and testnet when releasing patches.